Tapptitude/Helpers/TTTableViewUpdater.swift

```diff
@@ -79,8 +104,8 @@ class TableViewUpdater: TTTableViewUpdater {
     
     // MARK: - items operation
     func tableView(_ tableView: UITableView, didUpdateItemsAt indexPaths: [IndexPath]) {
-        batchOperation?.append({
-            tableView.reloadRows(at: indexPaths, with: .automatic)
+        batchOperation?.append({[weak self] in
```

Comment:
if we're to say 

let config = animationConfig
batchOperation?.append({ 
  tableView.reloadRows(at: indexPaths, with: config.itemsReload)
})

will capture config value at the moment when didUpdateItemsAt is called